app/scripts/directives/resources.js

```diff
@@ -24,10 +71,18 @@ angular.module('openshiftConsole')
         imagesByDockerReference: '=',
         builds: '=',
         detailed: '=?',
+        initContainers: '=?',
```

Comment:
Appears the property ```detailed``` is also used for the ```health checks if missing``` info alert... adjusting the ```initContainers``` as part of details now affects the banner display as well.